gpt4all-chat/qml/ModelSettings.qml

```diff
@@ -452,6 +452,50 @@ MySettingsTab {
                 Accessible.name: topPLabel.text
                 Accessible.description: ToolTip.text
             }
+            MySettingsLabel {
+                id: minPLabel 
+                text: qsTr("Min P")
+                Layout.row: 3
+                Layout.column: 0
+            }
+            MyTextField {
+                id: minPField
+                text: root.currentModelInfo.minP
+                color: theme.textColor
+                font.pixelSize: theme.fontSizeLarge
+                ToolTip.text: qsTr("Sets the minimum probability value for a token to be considered.")
```

Comment:
Actually, this would be more accurate:
```suggestion
                ToolTip.text: qsTr("Sets the minimum relative probability for a token to be considered.")
```